.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1616,6 +1789,18 @@ iam_DeleteUser:
   synopsis: delete an &IAM; user.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DeleteUserExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DeleteResourcesAsync
```

Comment:
These two examples are almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
Followed your suggestion.